storybookjs/storybook
# Added "key" prop to list items inside test failure message.

Issue:
React complains about missing "key" prop in lists displaying a failed Jest test.

## What I did
Added "key" prop equal the list element's index on the list.

## How to test
Create a failing Jest test, let jest output results to a JSON file and have that file load be loaded by Storybook's Jest addon.

Is this testable with jest or storyshots?
Jest's output JSON file containing information about a failed test is required.

Does this need a new example in the kitchen sink apps?
No

Does this need an update to the documentation?
No

~If your answer is yes to any of these, please make sure to include it in your PR.~
